sentry/src/test/java/io/sentry/SentryOptionsTest.kt

```diff
@@ -91,11 +91,6 @@ class SentryOptionsTest {
         assertFailsWith<IllegalArgumentException> { SentryOptions().sampleRate = -0.0000000000001 }
     }
 
-    @Test
-    fun `when setSampling is set to exactly 0, setter throws`() {
```

Comment:
maybe add a test `when setSampleRate is set to exactly 0, value is set` like it's done for `setTracesSampleRate`?

Reply:
We already test this as part of `SampleRateUtilsTest`, and in the options we just use the same logic, so I'd keep it like that if you don't mind